src/test/java/com/gooddata/report/ReportExportFormatTest.java

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2004-2017, GoodData(R) Corporation. All rights reserved.
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE.txt file in the root directory of this source tree.
+ */
+package com.gooddata.report;
+
+import org.testng.annotations.Test;
+
+import java.util.Arrays;
+
+import static com.gooddata.report.ReportExportFormat.toExportFormat;
+
+@SuppressWarnings("deprecation")
+public class ReportExportFormatTest {
+
+    @Test(expectedExceptions = IllegalArgumentException.class)
+    public void shouldFailOnNullInput() throws Exception {
+        toExportFormat(null);
+    }
+
+    @Test
+    public void shouldConvertAllValues() throws Exception {
+        Arrays.stream(ReportExportFormat.values())
+                .forEach(ReportExportFormat::toExportFormat);
```

Comment:
What about to assert results? :)

Reply:
wel this is more about not throwing an exception for all the cases